layouts/shortcodes/agile-pentest.md

```diff
@@ -0,0 +1 @@
+An Agile Pentest focuses on code changes or a specific area of an asset and comes with an Automated Report intended for internal use
```

Comment:
There are some additional use cases if you want to capture here.
https://docs.google.com/presentation/d/1Rujaf_XSwW5VdiSHrYuecNjAcrmobx97jYI5SM9NLrg/edit#slide=id.g1431c57e796_4_1

Reply:
I've set up DOCS-301. Great idea, I love including use cases, but I want us to be able to Merge without delay on Monday, as scheduled.

Reply:
@mjang-cobalt The use cases listed in the presentation are captured in `pentest-types.md`.

Reply:
Quite right, thanks.

<img width="1159" alt="Screen Shot 2022-10-03 at 6 16 58 AM" src="https://user-images.githubusercontent.com/84352037/193586991-ac5121bc-f66c-4328-8d64-45d7400982f1.png">